docs/ops/arithmetic/Tanh_1.md

```diff
@@ -2,24 +2,50 @@
 
 **Versioned name**: *Tanh-1*
 
-**Category**: *Activation function*
+**Category**: *Arithmetic function*
 
 **Short description**: Tanh element-wise activation function.
```

Comment:
The description still says that it is activation function...

Reply:
`Activation` is drop and `Arithmetic` is set or I miss something?

Reply:
Line 7:

\*\*Short description\*\*: Tanh element-wise **activation** function.

Reply:
Now I see. Thanks for clarification. Fixed.